PcapDotNet/src/PcapDotNet.Packets/Dhcp/Options/DhcpARPCacheTimeoutOption.cs

```diff
@@ -26,6 +26,7 @@ public DhcpArpCacheTimeoutOption(uint time) : base(time, DhcpOptionCode.ArpCache
         {
         }
 
+        [DhcpOptionReadRegistration(DhcpOptionCode.ArpCacheTimeout)]
```

Comment:
Any reason not to have this attribute on the classes instead of the creation method?
As long as you don't have two creation methods for the same class, I think putting this on the class is easier to maintain.